tls/s2n_security_policies.c

```diff
@@ -37,6 +37,7 @@ const struct s2n_security_policy security_policy_20190801 = {
      * name because customers use it.
      */
     .signature_preferences = &s2n_signature_preferences_20200207,
+    .certificate_signature_preferences = &s2n_certificate_signature_preferences_20201110,
```

Comment:
Unfortunately, you can't change the behavior of an existing security policy. If someone is using 20190801 by name, you'll break our promise not to change their behavior. You'll need to create a new security policy and change "default_tls13" to point to that.